orchestrator/orchestrator.js

```diff
@@ -0,0 +1,62 @@
+const request = require('request-promise')
+const logger = require('winston')
+const habanero = require('node-red-habanero')
+const habaneroSettings = require('../node_modules/node-red-habanero/settings')
+
+const config = require('../config/server')
+
+const HABANERO_ADMIN_ROOT = '/orchestrator'
+const HABANERO_NODE_ROOT = '/orchestrator-api'
+
+function getHabaneroBaseUrl(req){
```

Comment:
Do you use this function somewhere else than inside the `habaneroBaseUrl `? I think you can just put the return value right there: 
```
const habaneroBaseUrl = `${req.protocol}://${req.get('host')}${HABANERO_ADMIN_ROOT}/`
```